### ISSUE TITLE
[Dashboard] Clicking 'Customize' on widgets when Dev Home Window size is smaller leads to inability to interact with the app

### ISSUE THREAD

**Issue by mathewleeza** [microsoft/devhome#584]
### Steps to reproduce the bug

Description
1. Shrink Dev Home to the smallest allowable size (does not have to be this size, but this size should guarantee replication behavior)
2. Navigate to a pinned GitHub Issues widget
3. Click three dots in top right corner --> click customize
4. A pop up appears to customize the widget, but I am unable to interact with it (close out prompt, pin widget, etc.) and I can no longer interact with the application

Expectation --> I should be able to launch the customization workflow regardless of window size of Dev Home on the screen

Reality --> Customization workflow in this state causes the entire app to no longer be usable. I have to change the window size in order to continue.
